bin/start-systemtest

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+##
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=${1}
+readonly USER_PASSWORD=${2}
+readonly DISPATCHER_ADDRESS=${3}
+
+readonly SANDBOX=$(cd "$(dirname "${0}")" && pwd)
+readonly SUBMIT_TASK_SCRIPT=${SANDBOX}/submit-task
+readonly GET_TASK_BY_ID_SCRIPT=${SANDBOX}/get-task-by-id
+readonly GET_TASK_LINKS_SCRIPT=${SANDBOX}/get-task-links
+readonly SYSTEMTEST_MD5SUM_RESULTS_FILE=${SANDBOX}/../resources/systemtest-md5sum-results
+
+get_timestamp(){
+  TIMESTAMP=$(date +"%T")
+  echo "${TIMESTAMP}"
+}
+
+get_timesleep(){
+  local TEST_VERSION=${1}
+
+  if [ ${TEST_VERSION} == 'nop' ]; then
+    echo "30"
+  elif [ ${TEST_VERSION} == 'nop-fake-files' ]; then
+    echo "60"
+  elif [ ${TEST_VERSION} == 'nop-download-files' ]; then
+    echo "120"
+  else
+    echo "300"
+  fi
+}
+
+get_task_by_id(){
+  local ID=${1}
+  local TASK=$(bash ${GET_TASK_BY_ID_SCRIPT} ${USER_EMAIL} ${USER_PASSWORD} ${ID} ${DISPATCHER_ADDRESS})
+
+  echo "${TASK}"
+}
+
+get_dataset(){
+  local TASK=${1}
+  local DATASET=$(echo ${TASK} | jq '.dataset' | tr -d \" | tr '[:upper:]' '[:lower:]')
+
+  echo "${DATASET}"
+}
+
+get_state(){
+  local	TASK=${1}
+  local STATE=$(echo ${TASK} | jq '.state' | tr -d \" | tr '[:upper:]' '[:lower:]')
+
+  echo "${STATE}"
+}
+
+check_situation(){
+  local ID=${1}
+  local TEST_VERSION=${2}
+  local TASK=$(get_task_by_id ${ID})
+  local STATE=$(get_state "${TASK}")
+  local DATASET=$(get_dataset "${TASK}")
+
+  echo "($(get_timestamp)) [${TEST_VERSION}] Checking task situation [${ID}]..."
+
+  if [[ ${TEST_VERSION} == 'endtoend-test' && ${DATASET} == 'landsat_7' ]]; then
```

Comment:
I didn't understand this landsat_7 case

Reply:
SAPS currently does not seek to know if the image of the task exists before adding it to the Catalog, it passes this responsibility to the inputdownloading step (if it downloads it is because it exists, otherwise the error may be related to the non-existence of the image), in other words, in a new submission of tasks to SAPS, from day X to day Y, it will add a task for each day, still being able to have more than one per day depending on how many LANDSAT programs are in effect in the year of the task , for example, 2015-06-23, two tasks would be added, one with LANDSAT_7 and the other LANDSAT_8. In this case, there is only LANDSAT_8 exists, causing LANDSAT_7 to fail in the inputdownloading phase (perhaps the date of the image in the test should be added to further specify this case).
There is still no issue to fix this problem.